final_task/pycalc/constants_and_operations.py

```diff
@@ -0,0 +1,27 @@
+import operator
+
+from pycalc.module_appender import (append_module_by_name)
+
+OPERATIONS = {'^': operator.pow,
```

Comment:
It's recommended to leave firt row empty in multiline statements
```suggestion
OPERATIONS = {
    '^': operator.pow,
```